README.md

```diff
@@ -207,15 +191,20 @@ Se asume que la persona tiene conocimientos previos en:
 		then().
 			statusCode(201).
 			log().all();
-				
 	}
     ```
-    Primero preparamos la request que enviaremos como un Json, para esto usamos la clase JSONObject y después imprimimos como se veria ese json que creamos.
+    Primero preparamos la request que enviaremos como un JSON, para esto usamos la clase JSONObject y después imprimimos como se vería ese JSON que creamos.
     Luego especificamos la url base (baseURI) a la cual le enviaremos el request.
-    Finalmente en formato gherkin preparamos, enviamos y validamos el request. Aquí ponemos los header necesarios y el body a enviar, luego la acción que se sería la url base y adicionamos el resto del endpoint para el método post, con el then verificamos el status code de la petición e imprimimos lo que nos retorno el endpoint.
+    Finalmente en formato gherkin preparamos, enviamos y validamos el request. Aquí ponemos los header necesarios y el body a enviar, luego la acción que sería la url base y adicionamos el resto del endpoint para el método post, con el then verificamos el status code de la petición e imprimimos lo que nos retornó el endpoint.
     La parte importante aquí es la acción (when) para especificar el método HTTP.
+
     __Nota:__ Si quisieramos imprimir (por debuguear rápidamente por ejemplo) como esta el request formado, podemos usar `System.out.println(request.toJSONString());`
 
+##### Ejercicio opcional:
+
+ Usando usando esta API de Pokemom ([pokeapi.co]( https://pokeapi.co)), consumir el endpoint GET para obtener la información de un pokemon
+ e imprimir el JSON de respuesta en la consola
+
 #### Creando peticiones PUT, PATCH and DELETE
 1. Crear el archivo `PutPatchDeleteExample.java` en la carpeta de `test`
```

Comment:
As you suggest in line 144, please change to "1. Crear la clase `PutPatchDeleteExample.java` en la carpeta de `test`" and any other that might be left